Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -579,17 +579,24 @@ private void RestoreList(string Name, IObservableList v, bool isLocalBackup = fa
 
         public bool RestoreFromBackup(bool isLocalBackup = false, bool clearBackup = true)
         {
+            bool isRestored = false;
             try
             {
-                return RestoreBackup(isLocalBackup);
+                isRestored = RestoreBackup(isLocalBackup);
+            }
+            catch(Exception exc)
+            {
+                Reporter.ToLog(eLogLevel.DEBUG, "Error occured in the Undo Process", exc);
```

Comment:
this should not be "Debug" error but "Error" level, "Debug" is only for extra info for the developer- here we have Error which needs to be recorded